components/salsa-macros/src/update.rs

```diff
@@ -35,7 +53,7 @@ pub(crate) fn update_derive(input: syn::DeriveInput) -> syn::Result<TokenStream>
                 .bindings()
                 .iter()
                 .fold(quote!(), |tokens, binding| quote!(#tokens #binding,));
-            let make_new_value = quote_spanned! {variant.ast().ident.span()=>
+            let make_new_value = quote! {
```

Comment:
why remove the span? confuses ra?

Reply:
Yes, attaching all your tokens to spans randomly has less than desireable effects. The tokens in this quote! are all meaningless so there is no need to give them a specific input span